utils/appliance.py

```diff
@@ -1364,6 +1364,18 @@ def get_yaml_config(self, config_name):
     def set_yaml_config(self, config_name, data_dict):
         return db.set_yaml_config(config_name, data_dict, self.address)
 
+    def set_session_timeout(self, timeout=86400):
+        try:
+            vmdb_config = self.get_yaml_config("vmdb")
+            if vmdb_config["session"]["timeout"] < timeout:
```

Comment:
`<` made sense in the original implementation, when we had a high degree of certainty that we were acting on a new appliance. Adding this as a method to `IPAppliance` opens it up to be used at any time, and should therefore use `!=`.